examples/dotnet/bin/README.md

```diff
@@ -0,0 +1,6 @@
+Please first run
```

Comment:
Do you still need to run these if we're checking the files in?

Reply:
those were just notes describing how to these files got there (without them, nobody would really know the specific way we generated these files).  I'll remove this readme.md but add in a snippet about it into the main readme

Reply:
SGTM